src/main/java/examples/Examples.java

```diff
@@ -172,4 +174,21 @@ public void example6(Vertx vertx) {
           return "Hello";
         });
   }
+
+  public void example7(Vertx vertx) {
+    // Create the circuit breaker as usual.
+    CircuitBreaker breaker = CircuitBreaker.create("my-circuit-breaker", vertx);
+    CircuitBreaker breaker2 = CircuitBreaker.create("my-second-circuit-breaker", vertx);
+
+    // Create a Vert.x Web router
+    Router router = Router.router(vertx);
+    // Register the metric handler
+    router.get("/metrics").handler(HystrixMetricHandler.create(vertx));
```

Comment:
Minor: maybe we should use a more specific name in the example and doc for the the endpoint? Like `/circuit-breaker` or `\hystrix-metrics`. Note that I understand this is user-configurable and that the default is not `/metrics` :)